manifests/eap/client.pp

```diff
@@ -41,6 +41,11 @@
 #   The destination file path inside the JBoss module to put the SSL
 #   certificate authority file. Defaults to 'cacert.pem'.
 #
+# [*ca_certificates*]
```

Comment:
I added the ssl_ prefix to ca_certificates to be consistent with the other variables.  

As far as deprecating ssl_ca_file_path, I wanted to leave the option for a consumer to do a simple setup (i.e. one CA cert, and one client cert) or more complex configurations with multiple CAs and/or client certs.